doc/Iothub_sdk_options.md

```diff
@@ -0,0 +1,69 @@
+# IoTHub C SDK Options
+
+This document describes the options available in the c sdk and how to use the.
+
+- [Setting an Options](#set_option)
+- [IoThub Client Options](#IotHub_option)
+- [Transport Options](#transport_option)
+
+<a name="set_option"></a>
+
+## Setting an Option
+
+Setting and option in the c-sdk is dependant on which api set you are using.  If using the convience layer you would set the options in the api as follows:
```

Comment:
Setting and option in the C-SDK is dependent on the API set you use.
If you are using the convenience layer you would set the options in the API as follows:

Reply:
What if the user is not using convenience layer?